ui/app/AppLayouts/Communities/controls/Options.qml

```diff
@@ -31,7 +31,7 @@ ColumnLayout {
         StatusCheckBox {
             id: archiveSupportToggle
             width: (parent.width-12)
-            checked: true
+            checked: false
```

Comment:
What if the user enables it here, but it's disabled under Advanced? I think this should rather read `enabled: advancedStore.archiveProtocolEnabled`, or even `visible: ...`

Reply:
Those two are different settings. The one for communities is for the community itself. So if it's OFF, even if users have it ON, they will never receive any magnet link, because the community doesn't have it.

On the other hand, if this setting is ON, then the community will send magnet links, but if the user has their Advanced setting OFF' then won,t process it.

Both need to be ON for the archive to work. the changes I did here is for new communities and new users to start at OFF.

Reply:
It's probably not a big deal, but really this checkbox will make the community owner think that the archive publishing will automatically work. But it's essential for the community owner to enable the new advanced setting as well.

https://github.com/status-im/status-go/blob/0ef4fb8cc73187ffd6f6552d48cd953957e09712/protocol/messenger_communities.go#L2280-L2282

Reply:
Done. I added a tooltip explaining that the advanced setting is needed. I tried making it show when the setting is OFF, but it was quite a pain to pain the store value, so I just show it always.